website/blog/2023-05-04-generating-dynamic-docs.md

```diff
@@ -0,0 +1,370 @@
+---
+title: "Accelerate your documentation workflow: Generate docs for whole folders at once"
+description: "For columns that are reused across models, Mikael walks through a DRY-method to make documentation easier, using the dbt Codegen package and docs blocks."
+slug: generating-dynamic-docs-dbt
```

Comment:
Something like bulk-docs-generation? @KiraFuruichi I assume you'll have specific ideas on a search-friendly slug